src/aleph/vm/hypervisors/firecracker/microvm.py

```diff
@@ -237,7 +238,7 @@ async def start_jailed_firecracker(self, config_path: Path) -> asyncio.subproces
             "--gid",
             gid,
             "--chroot-base-dir",
-            JAILER_BASE_DIRECTORY,
+            str(self.jailer_base_directory),
```

Comment:
Why str() instead of .name ?

Reply:
It's using str() for `firecracker_bin_path`  and `self.jailer_bin_path` just above so it's consistent.

And frankly, I always used str() for Path because and I didn't really know `.name` existed before.
If you prefer to standardize one one or the other it's okay for me I will adapt.
But with str() I know it just work to convert a path to as str, for `.name` I couldn't tell without checking if it's just going to return the filename of the leaf or the whole path, a string or something else.